packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -113,10 +118,12 @@ describe('TransactionResponse', () => {
   });
 
   it('should ensure getTransactionSummary fetchs a transaction and assembles transaction summary', async () => {
-    const { ip, port } = await launchNode({
-      args: ['--poa-instant', 'false', '--poa-interval-period', '1s'],
+    using launched = await launchTestNode({
```

Comment:
This test was missing a `cleanup` call at the end so I switched to using `launchTestNode` right away.